.github/workflows/security-scan.yml

```diff
@@ -1,14 +1,17 @@
 name: Security Scan
 
 on:
+  push:
+    branches:
+      - upgrade/go-1-22-4
```

Comment:
Do we want to remove this?

Reply:
Looks like you are still testing. Other then this, everything looks good.

Reply:
Yeah, sorry I should've removed the review request when I decided to do further testing.